docs/topics/serialization.txt

```diff
@@ -347,6 +347,37 @@ again a mapping with the key being name of the field and the value the value:
 
 Referential fields are again represented by the PK or sequence of PKs.
 
+.. _custom-serialization-formats:
+
+Custom serialization formats
+----------------------------
+
+In addition to the above formats, you can create your own serialization format.
+
+First, define a ``Serializer`` and a ``Deserializer`` class. These can override
+existing serialization format classes::
+
+    from django.core import serializers
+
+
+    class Serializer(serializers.python.Serializer): ...
+
+
+    class Deserializer(serializers.python.Deserializer): ...
+
+Then add the module containing the serializer definitions to your
+:setting:`SERIALIZATION_MODULES` setting::
+
+    SERIALIZATION_MODULES = {
+        "custom": "path.to.custom_serializer",
```

Comment:
This path here might be confusing if it is not defined above what it is. Maybe we could define above that the code goes in a python file with a certain name and then call it here to help understand that this is the path to the python file (or module) that contains the serializers.

Reply:
Yes! @AMK9978 something like
```
.. code-block:: python
    :caption: ``path/to/custom_serializer.py``
```
should work

Reply:
Yes, exactly.